pkgs/tools/admin/gam/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  fetchFromGitHub,
+  python3,
+  lib,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "gam";
+  version = "6.21";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "GAM-team";
+    repo = "gam";
+    rev = "v${version}";
+    sha256 = "sha256-MNLjdn0YCZ2nasMyuRM72VcGMTbr6jIQyj29+MV1OjQ=";
+  };
+
+  sourceRoot = "source/src";
+
+  patches = [
+    # Also disables update check
+    ./signal_files_as_env_vars.patch
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    dateutil
+    distro
+    filelock
+    google-api-python-client
+    google-auth
+    google-auth-oauthlib
+    passlib
+    pathvalidate
+    setuptools
+  ];
+
+  # Use XDG-ish dirs for configuration. Maybe more than these needed.
+  makeWrapperArgs = [
+      "--set GAMUSERCONFIGDIR '~/.config/gam'"
```

Comment:
I'm not exactly sure how to properly handle this case, but `XDG_CONFIG_HOME` could be something other than `~/.config` and etc.

Reply:
Used `--set-default` instead. I would like to use `XDG_CONFIG_HOME` and `XDG_CACHE_HOME`, but cannot find a simple way to do so.